crates/starknet-devnet/tests/test_get_events.rs

```diff
@@ -151,4 +151,44 @@ mod get_events_integration_tests {
 
         get_events_correct_chunking(&devnet, true).await
     }
+
+    #[tokio::test]
+    async fn get_events_errors() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+
+        let chunk_size = 3;
```

Comment:
Instead of casting below, why not define the correct type right away:
```suggestion
        let chunk_size = 3_u64;
```

Reply:
my bad